src/components/ConcurrencyLimitsV2Table.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <ConcurrencyLimitsPageEmptyState v-if="empty && loaded" />
+  <p-table v-else class="concurrency-limits-table__table" :columns="columns" :data="concurrencyLimits">
+    <template #tag="{ row }">
+      <p-link :to="routes.concurrencyLimit(row.id)">
+        {{ row.name }}
+      </p-link>
+    </template>
+    <template #active-task-runs="{ row }">
+      {{ row.activeSlots }}
+    </template>
+
+    <template #action-heading>
+      <span />
+    </template>
+
+
+    <template #empty-state>
+      <PEmptyResults>
+        <template #message>
+          No task concurrency limits
+        </template>
+      </PEmptyResults>
```

Comment:
```suggestion
      <p-empty-results>
        <template #message>
          No task concurrency limits
        </template>
      </p-empty-results>
```
We try to use kebab-casing for components that are installed globally like those coming from prefect-design